samples/snippets/src/main/java/com/example/spanner/QueryProtoColumnSample.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2022 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.spanner;
+
+import com.example.spanner.SingerProto.Genre;
+import com.example.spanner.SingerProto.SingerInfo;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.Statement;
+
+public class QueryProtoColumnSample {
```

Comment:
This sample reads data from Proto columns and Array of Proto columns. However it is not conveyed in the name.
Can we rename it or can we add a comment stating this?